launcher/ui/dialogs/AboutDialog.cpp

```diff
@@ -111,7 +111,7 @@ QString getCreditsHtml()
     stream << "<p>TakSuyu &lt;<a href='mailto:taksuyu@gmail.com'>taksuyu@gmail.com</a>&gt;</p>\n";
     stream << "<p>Kilobyte &lt;<a href='mailto:stiepen22@gmx.de'>stiepen22@gmx.de</a>&gt;</p>\n";
     stream << "<p>Rootbear75 &lt;<a href='https://twitter.com/rootbear75'>@rootbear75</a>&gt;</p>\n";
-    stream << "<p>Zeker Zhayard &lt;<a href='https://twitter.com/zeker_zhayard'>@Zeker_Zhayard</a>&gt;</p>\n";
+    stream << QString("<p>ZekeZ %1</p>\n").arg(getGitHub("ZekeZDev"));
```

Comment:
This person is not ZekeZ

Reply:
https://github.com/PrismLauncher/PrismLauncher/blob/0700c76c361fa59396cca815103054c7395353eb/launcher/ui/dialogs/AboutDialog.cpp#L80 this person is

Reply:
ups